packages/librsvg.rb

```diff
@@ -1,50 +1,43 @@
 require 'package'
 
 class Librsvg < Package
-  description 'Scalable Vector Graphics (SVG) rendering library.'
+  description 'SVG library for GNOME'
   homepage 'https://wiki.gnome.org/Projects/LibRsvg'
-  version '2.42.3'
+  version '2.50.1'
   compatibility 'all'
-  source_url 'https://download.gnome.org/sources/librsvg/2.42/librsvg-2.42.3.tar.xz'
-  source_sha256 '704f2c44b9b170fc5498de36a161d01ca8f584ba9c42654b98565a7b7bcbe657'
+  source_url 'https://ftp.gnome.org/pub/GNOME/sources/librsvg/2.50/librsvg-2.50.1.tar.xz'
+  source_sha256 '9bd9799322e06cf5db19b9f7afb728edac6efcf0110baafc44f0f96f45df9a09'
```

Comment:
Why the source location change?

Reply:
I think they are interchangeable. Just the first URL I found when searching for updates when building.

```https://download.gnome.org/sources/librsvg/2.50/librsvg-2.50.1.tar.xz``` works just as well. I'll change it so there is no change in source location.